test/test_signature_information.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from freeclimb.utils.signature_information import SignatureInformation
+
+class TestSignatureInformation(unittest.TestCase):
+    """SignatureInformation unit test stubs"""
+
+    def setUp(self):
+        requestHeader = "t=1679944186,v1=c3957749baf61df4b1506802579cc69a74c77a1ae21447b930e5a704f9ec4120,v1=1ba18712726898fbbe48cd862dd096a709f7ad761a5bab14bda9ac24d963a6a8"
+        self.model = SignatureInformation(requestHeader)
```

Comment:
If we look at the unittest docs for organizing test blocks: https://docs.python.org/3/library/unittest.html#organizing-test-code

They have a setup and teardown section, and then they have individual test cases, I just followed this pattern for testing in python

Reply:
I can add an instance test but because we're testing functions in this file and the request verifier test file, I wasn't sure if an instance test was needed

Reply:
the link shows that its using a specific naming pattern, `self.widget = new Widget()`.
in this case, make it specific to the `SignatureInformation` class.

Reply:
will edit this in latest commit